js/payment-methods/monero.js

```diff
@@ -92,8 +94,8 @@ app.paymentMethods.monero = (function() {
 				},
 				type: 'text',
 				required: true,
-				validate: function(value) {
-					this.validatePrivateViewKey(value);
+				validate: function(value, data) {
+					this.validatePrivateViewKey(value, data.publicAddress);
```

Comment:
The key of the property that contains the "publicAddress" is "monero.publicAddress", so it seems it should be as below:

```javascript
var publicAddress = data[this.ref + '.publicAddress'];
this.validatePrivateViewKey(value, publicAddress);
```

Reply:
Good catch! Fixed now :)